python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -3,5 +3,8 @@
 def home(request):
     return render_to_response('home.html')
 
+def map(request):
```

Comment:
You know, I don't have a strong opinion. Seeing as there are going to be only a few html based views and maybe an API endpoint or two, function based may be a simpler choice?